src/server/tablet_snap.rs

```diff
@@ -119,7 +119,7 @@ async fn send_snap_files(
     let mut total_sent = msg.compute_size() as u64;
     let mut chunk = SnapshotChunk::default();
     chunk.set_message(msg);
-    chunk.set_data(usize::to_ne_bytes(SNAP_CHUNK_LEN).to_vec());
+    chunk.set_data(usize::to_be_bytes(SNAP_CHUNK_LEN).to_vec());
```

Comment:
Should use le_bytes instead. And there are multiple places that is using ne_bytes.

Reply:
fixed

Reply:
No, there should be 3 places.